crd-generator/test/pom.xml

```diff
@@ -90,6 +90,14 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <compilerArgs>
+            <arg>-Aio.fabric8.crd.generator.parallel=false</arg>
```

Comment:
this should be redundant as it's already the default, isn't it?

Reply:
yes it is. I wanted to make it explicit and leave it there as example since it's not trivial to add processor options

Reply:
👍